packages/vsce/src/SshClientCache.ts

```diff
@@ -28,8 +32,12 @@ export class SshClientCache extends vscode.Disposable {
         }
     }
 
+    public static initialize(context: vscode.ExtensionContext): SshClientCache {
+        SshClientCache.mInstance = new SshClientCache(context);
```

Comment:
I can see how it might be easier to manipulate (and test) the instance when it's initialized via a function instead of a getter. I do have one question... What about keeping the "singleton" approach?

```suggestion
        SshClientCache.mInstance ??= new SshClientCache(context);
```

And perhaps making this function `@internal` 😋

Reply:
Discussed in person, I believe this method is fine to leave as-is since we need to pass `context`, and we don't need to mark it internal since this class is not exported.